.gitignore

```diff
@@ -146,6 +146,7 @@ tagCountHistogram2.png
 uniqueHistogram.png
 test-files/cmd/current
 test-files/DcTigerRoadsHighwayExactMatchInputs.osm
+test-files/DcTigerRoadsFrechetInput.osm
```

Comment:
It isn't different, it is just a copy because it does the exact highway match.  The script makes a copy of the file so that it can use two different files as input just like the `test-files/cmd/slow/ConflateCmdHighwayExactMatchInputsTest.sh` test that I duplicated and added the setting to force it to use Frechet.